onetl/connection/db_connection/mongodb/connection.py

```diff
@@ -504,6 +504,16 @@ def write_df_to_target(
             else "append"
         )
 
+        if self._collection_exists(target):
+            if write_options.if_exists == MongoDBCollectionExistBehavior.ERROR:
+                raise ValueError("Operation stopped due to if_exists set to 'error'.")
```

Comment:
```suggestion
                raise ValueError("Operation stopped due to 'if_exists' set to 'error'.")
```

ещё бы я указал где именно сидит параметр if_exists потому что в потоке лога скорее всего будет вообще не понятно где что нужно поменять и где искать if_exists. было бы удобно сразу подсказать где он.